mobile/native/src/api.rs

```diff
@@ -154,3 +158,8 @@ pub fn create_invoice() -> Result<String> {
 pub fn send_payment(invoice: String) -> Result<()> {
     ln_dlc::send_payment(&invoice)
 }
+
+pub fn insert_last_login() -> Result<LastLogin> {
+    let last_login = db::update_last_login()?;
```

Comment:
nit: You could write this in one line.

```
db::update_last_login()
```

Also it looks strange that the api is named `insert_last_login` while the db function is named `update_last_login`. Why not use the same name?

Reply:
> Why not use the same name?

Because of reasons! 😬